examples/dreambooth/train_dreambooth.py

```diff
@@ -794,6 +792,7 @@ def encode_prompt(text_encoder, input_ids, attention_mask, text_encoder_use_atte
     prompt_embeds = text_encoder(
         text_input_ids,
         attention_mask=attention_mask,
+        return_dict=False,
     )
```

Comment:
```suggestion
    prompt_embeds = text_encoder(text_input_ids, attention_mask=attention_mask, return_dict=False)
    )
```

Will probability fit within the permissible line-length.

Reply:
Sure!